GRDB/Core/Database.swift

```diff
@@ -317,11 +269,30 @@ extension Database {
 extension Database {
 
     // MARK: - Database Setup
-    
+    func setupConnection() throws {
+        do {
```

Comment:
maybe there should be a new "call only once" sanity check?

```
private var isSetup: Bool = false
func setupConnection() throws {
    assert(isSetup == false)
    ...
    isSetup = true
}
```

Reply:
No, we don't have to.